hs628_tools/Test.py

```diff
@@ -0,0 +1,86 @@
+
+import numpy as np
+from sklearn import datasets
+import csv
+import os
+
+
+
+
+class data_split:
+    """
+    Create class data_split to predict a perfect train and test data split ratio
+
+    define methods to accomplish this task
+
+    return a split ratio
+
+    """
+    def __init__(self, data):
+        self.data = data
+
+    #set path to read the csv file in the directory
+    def test_train():
+
+        pathName = os.getcwd()
+
+        dataset = []
+        fileNames = os.listdir(pathName)
+
+        # read file with extension .csv
+        for fileNames in fileNames:
+            if fileNames.endswith(".csv"):
+                dataset.append(fileNames)
+
+        for k in dataset:
```

Comment:
If you want to clean all the CSV files in that directory, you'll need to save each dataset in a list and then proceed with the cleaning part. The current code is looping through the CSV files and read the file and save it into 'data' variable and each time the 'data' will be entirely replaced with a new CSV file content. Therefore, you are only saving the very last CSV file in your folder.

Reply:
That is correct. This part of the code is taking in csv file and reading it in. Looping through each data point is the next step. I hope you saw that as well. Please let me know if you didn't.